doc/explanation/instance_config.md

```diff
@@ -21,14 +21,26 @@ Instance options
 
   In the YAML configuration, options are located under the `config` entry.
 
-  See {ref}`instance-options` for a reference of available instance options.
+  See {ref}`instance-options` for a reference of available instance options, and {ref}`instances-configure-options` for instructions on how to configure the options.
+
+Instance devices
+: Instance devices are attached to an instance.
+  They include, for example, network interfaces, mount points, USB and GPU devices.
+  Devices are usually added after an instance is created with the `lxc config device add` command, but they can also be added to a profile or a YAML configuration file that is used to create an instance.
+
+  Devices can have instance device options, depending on the type of the instance device.
```

Comment:
This reads a bit "robotic" (specifically the repetition of "instance device"), perhaps something along the lines of would be clearer:

> Each type of instance device has its own specific set of options.